index.js

```diff
@@ -346,6 +347,13 @@ module.exports = {
   },
 
   postBuild(result) {
+    // Need to update manifest from html file.
+    // Set environment variable `FASTBOOT_HTML_MANIFEST` to `true`
+    // Usage `FASTBOOT_HTML_MANIFEST=true ember s`
+    if(process.env.FASTBOOT_HTML_MANIFEST === 'true') {
+      updateManifestFromHtml(result.directory);
+    }
```

Comment:
In order for this to work correctly internally, we need to make sure our placeholder addon runs before fastboot.